portfolio/src/main/java/com/google/sps/data/LoadFollowedTags.java

```diff
@@ -28,7 +28,7 @@
 import java.util.List;
 
 public final class LoadFollowedTags {
-  
+  public static String tagQuery = "followedTag";
```

Comment:
**nit:** i think it should be a constant and in caps like final static String TAG_QUERY...

Reply:
Actually you could do this in the Servlet so that you don't need another declaration here